src/ddmd/expression.d

```diff
@@ -4210,6 +4210,13 @@ extern (C++) final class DsymbolExp : Expression
             return e;
         }
 
+        if(ScopeDsymbol scs = s.isForwardingScopeDsymbol())
```

Comment:
This was needed for the `__previous` special symbol. It can be safely removed. (I'll do that.)